docs/pages/development-bundles.md

```diff
@@ -0,0 +1,93 @@
+@page StealJS.guides.development_bundles Development Bundles
+@parent StealJS.guides 2
+@outline 0
+
+@body
+
+<!-- hack! -->
+<style>.contents { display: none; }</style>
+
+In this guide we'll go through the steps required to set up `steal-tools` development bundles and
+speed up your application development process.
```

Comment:
I would add a note in here (like a blockquote) that says that we are assuming that the reader already has a project setup. Maybe refer them to the progressive loading guide if they don't already have a project to try this with.